src/shutdown_manager.ts

```diff
@@ -36,8 +36,8 @@ export default class ShutdownManager {
         return true;
     }
 
-    async getShutdownStatus(ctx: Context, details: InstanceDetails): Promise<boolean> {
-        const key = this.shutDownKey(details.instanceId);
+    async getShutdownStatus(ctx: Context, instanceId: string): Promise<boolean> {
```

Comment:
Should probably also change the setShutdownStatus method to only accept instanceId